routes/users.router.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const { PlayList } = require('../models/playList.model');
+const router = express.Router();
+const { User } = require('../models/user.model');
+
+router.get('/', (req, res) => {
+    res.json({ success: true, message: 'heya' });
+});
+
+// req.body: {fistName, lastName, email, password}
+// response {_id, firstName, lastName, email, ref(likedVideos), ref(playLists)}
+router.post('/', async (req, res) => {
+    try {
+        const newUser = req.body;
+
+        const user = new User(newUser);
```

Comment:
Here, it is not checking if the email id already exists in the DB or not. Hence while a new user is registering with the same email ID, the right error message is not being presented.

Reply:
I am returning the right error message now in the error key of the response, the error message will be accurate now